package-lock.json

```diff
@@ -89,6 +89,7 @@
         "stream-browserify": "3.0.0",
         "styled-components": "5.3.0",
         "superagent": "6.1.0",
+        "timezones.json": "1.5.2",
```

Comment:
This should be fixed if you update to the latest master. Git seems to be handling it fine though